content/en/blog/2023-11-modelix-platform-release-23.2/index.md

```diff
@@ -0,0 +1,89 @@
+---
+date: 2023-11-02
+title: "modelix platform release 23.2"
+---
+
+We are happy to announce the Autumn release of the modelix platform: **`modelix-23.2`**
+
+The version bundled in this release are:
+
+```
+modelixCoreVersion = 3.12.1
+incrementalVersion = 0.2.0
+buildToolsVersion = 1.1.0
+modelixMPSPatchVersion = 157
+```
+
+If you want to know how to use this platform in your Gradle build, check out the corresponding [Documentation](https://docs.modelix.org/modelix/main/platform/howto/usage-platform.html).
+
+- [Feature Highlights](#feature-highlights)
+  - [ModelQL](#modelql)
+  - [Model client as NPM package (alpha)](#model-client-as-npm-package-alpha)
+  - [Vue.js Bindings (alpha)](#vuejs-bindings-alpha)
+  - [MPS Model Adapters](#mps-model-adapters)
+  - [Bulk Model Synchronization Plugin (alpha)](#bulk-model-synchronization-plugin-alpha)
+- [Development Insights](#development-insights)
+  - [New approach to MPS related components](#new-approach-to-mps-related-components)
+  - [Contribution guide](#contribution-guide)
+  - [Marking experimental features](#marking-experimental-features)
+  - [Work on code quality](#work-on-code-quality)
+- [What's to come](#whats-to-come)
+
+# Feature Highlights
+
+## ModelQL
+
+The [ModelQL query language](https://docs.modelix.org/modelix/modelix-23.2/core/explanation/modelql.html) provides a dynamic way of loading parts of the model on demand while reducing the number of requests to a minimum.
+
+## Model client as NPM package (alpha)
```

Comment:
```suggestion
## Model client as NPM package (experimental/unstable)
```

To be consistent with the unstable annotation and news?

Reply:
Done (decided for "experimental")